<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProblemDialog</class>
 <widget class="QWidget" name="ProblemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="choseActionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chose action:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newPicButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>341</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Chose new picture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moreDensityButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>341</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>More information-density</string>
   </property>
  </widget>
  <widget class="QPushButton" name="morePicsButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>341</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Chose more pictures</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="xXPixelLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x/x Pixel</string>
   </property>
  </widget>
  <widget class="QLabel" name="bpPLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x Bit/Pixel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
